app/models/scaptimony/scap_content.rb

```diff
@@ -36,6 +37,7 @@ class ScapContent < ActiveRecord::Base
 
     validates_with Scaptimony::DataStreamValidator
     validates :title, :presence => true
+    validates :digest, :presence => true
```

Comment:
if I understand correctly, this causes digest method call which does the calculation. Tricky :-)